packages/starters/backend/sirius-components-starter/src/main/java/org/eclipse/sirius/components/starter/SiriusWebStarterConfiguration.java

```diff
@@ -130,7 +130,7 @@ public IEventProcessorSubscriptionProvider eventProcessorSubscriptionProvider(IE
             public Flux<IPayload> getSubscription(String editingContextId, IRepresentationConfiguration representationConfiguration, IInput input) {
                 return editingContextEventProcessorRegistry.getOrCreateEditingContextEventProcessor(editingContextId)
                         .flatMap(processor -> processor.acquireRepresentationEventProcessor(representationConfiguration, input))
-                        .map(representationEventProcessor -> representationEventProcessor.getOutputEvents(input))
+                        .map(representationEventProcessor -> representationEventProcessor.customize(representationEventProcessor.getOutputEvents(input)))
```

Comment:
Re-read the code I gave you, the representationEventProcessor should not be in charge of the customization, it should be another service, the customizer, instead.